.gitignore

```diff
@@ -1,4 +1,3 @@
 vendor
-composer.lock
```

Comment:
There are multiple views on this matter, mine and many others are "don't commit it for libs".